### PR TITLE
Include volumenameprefix in GlusterFS example

### PR DESCRIPTION
**Pull request by jboxman**
- https://bugzilla.redhat.com/show_bug.cgi?id=1667161

@openshift/team-documentation PTAL thanks

### CODE REVIEW

File: install_config/persistent_storage/topics/glusterfs_storageclass.adoc [hunk 1]
```diff
@@ -41,6 +42,7 @@ Volume Options] for more GlusterFS volume options.
 <9> Optional. The
 link:https://docs.gluster.org/en/v3/Quick-Start-Guide/Architecture/[type of
 volume] to use.
+<10> Optional. Enables custom volume name support using the following format: <volumenameprefixstring>_<namespace>_<claimname>_UUID.
```
Comment (kalexand-rh) [inline install_config/persistent_storage/topics/glusterfs_storageclass.adoc]: Enclose the whole sample in ``.
Can we reasonably expect the user know what volumenameprefixstring and claimname are? If not, explain where to obtain them.

**Review by kalexand-rh — COMMENTED** [review]
A pick and a question. Otherwise LGTM.

**Comment by jboxman** [comment]
@openshift/team-documentation, updated with additional clarity on the output provided when the `volumenameprefix` parameter is specified. PTAL.

File: install_config/persistent_storage/topics/glusterfs_storageclass.adoc [hunk 2]
```diff
@@ -41,6 +42,10 @@ Volume Options] for more GlusterFS volume options.
 <9> Optional. The
 link:https://docs.gluster.org/en/v3/Quick-Start-Guide/Architecture/[type of
 volume] to use.
+<10> Optional. Enables custom volume name support using the following format:
+`<volumenameprefix>_<namespace>_<claimname>_UUID`. If you create a new PVC
+called `myclaim` in your project `project1` using this storageClass, the PVC
+name will be `custom-project1-myclaim-UUID`.
```
Comment (liangxia) [inline install_config/persistent_storage/topics/glusterfs_storageclass.adoc]: s/the PVC name will be `custom-project1-myclaim-UUID`/the volume name will be `custom_project1_myclaim_UUID`/

Comment (jboxman) [reply install_config/persistent_storage/topics/glusterfs_storageclass.adoc]: Thanks, updated!

**Comment by liangxia** [comment]
LGTM (from QE). Thank you.

**Pull request merged**